variables.tf

```diff
@@ -99,18 +100,6 @@ variable "ssh_key_name" {
   default     = ""
 }
 
-variable "ami_id" {
```

Comment:
Removing this will break any customers that use a custom AMI

Reply:
yes, we are removing support for custom AMI ... btw, we have needed WG linux packages (distro-specific) since Netagent 1.40.

let's release this as a major TF update then.